services/manifest-repo-export-service/pkg/service/version_test.go

```diff
@@ -509,6 +509,17 @@ func TestGetManifests(t *testing.T) {
 				if err != nil {
 					return err
 				}
+				err = dbHandler.DBInsertRelease(ctx, transaction, db.DBReleaseWithMetaData{
```

Comment:
Because we now check for the latest version on the manifest, this release was missing from the database.

Reply:
In lines 475:493, We are adding two releases with `ReleaseNumber: 3`. I guess, if we just change one of them to `ReleaseNumber: 2`, that would work.